grails-app/services/com/unifina/service/StreamService.groovy

```diff
@@ -65,6 +65,9 @@ class StreamService {
 		Stream stream = new Stream(params)
 		stream.id = id
 		stream.config = params.config
+		if (stream.name == null || stream.name.trim() == "") {
```

Comment:
Default name is set in `Stream` already (is this code redundant?)

Reply:
params variable might override it. There is a test for it:  "createStream replaces empty name with default value"